src/character.h

```diff
@@ -1234,7 +1234,8 @@ class Character : public Creature, public visitable
         /** Handles bionic activation effects of the entered bionic, returns if anything activated */
         bool activate_bionic( int b, bool eff_only = false, bool *close_bionics_ui = nullptr );
         std::vector<bionic_id> get_bionics() const;
-        std::vector<item> get_pseudo_items() const;
+        std::vector<item> &get_pseudo_items() const;
```

Comment:
I don't think you should be able to freely modify the cached items.
```suggestion
        const std::vector<item> &get_pseudo_items() const;
```

Reply:
Changed it to return a `std::vector<const item*>`, this way we can keep permanent instances of the items and also allowed to add persistence of CBM items (serializing weapons into the savefile).